package.json

```diff
@@ -39,10 +40,12 @@
     "portfinder": "~1.0.10",
     "request": "~2.87.0",
     "requestretry": "~2.0.2",
+    "sauce-connect-launcher": "~1.3.1",
```

Comment:
@xwa130saucer @loyalchow are we going to fork and publish a new package before merging?

Reply:
That's a good question, we could either do this or contact with Bermi to ask him if he is willing to migrate his repo to `saucelabs` github repo and maintain there? Let's discuss in the huddle?

Reply:
Let's fork and publish (but in a different ticket). Migrating to sauce repo will take time - forking will allow us to move forward now

Reply:
Do you think `npm install` directly from `github` or we give another name and publish to npm?

Reply:
I addressed in this PR since I found out it's just one line change, is this OK?